app/main/views/g_cloud.py

```diff
@@ -236,7 +239,8 @@ def search_services():
         summary=search_summary.markup(),
         title='Search results',
         total=search_results_obj.total,
-        gcloud_framework_description=framework_helpers.get_framework_description(data_api_client, 'g-cloud'))
+        gcloud_framework_description=framework_helpers.get_framework_description(data_api_client, 'g-cloud'),
+        clearFiltersUrl=clear_filters_url)
```

Comment:
Let's stick to the precedent here and call this variable `clear_filters_url` on both sides.